js/colourpicker.js

```diff
@@ -668,160 +670,163 @@
+    // gets a flipped version of colsToNames so that we can search by colors for names
```

Comment:
got it. Editing. I don't have a reply box for the "test" question, but it was an accidental commit. I was testing out a command to open sublime over a network connection and created that file, then committed it one time accidentally. It shouldn't be in there now.